### ISSUE TITLE
Diagramms:  All busses

### ISSUE THREAD

**Issue by Bachibouzouk** [open-plan-tool/gui#219]
If a flow is zero, it should not be displayed
Flow in a bus should be positive and flow out negative by default

**Comment by mstich0** [comment]
agreed, also we wanted to stack all inputs and all outputs and extended the amount of colors

**Comment by Bachibouzouk** [comment]
Information needed to make this plot:
- input and output flows for all busses
- energy_vector of the busses
- rule to select which components are displayed as negative --> in are positive and out are negative

**Comment by mstich0** [comment]
just as a clarification: 
when we say "Flow in a **bus** should be positive and flow out negative by default"
this means in respect to a **component** that anything flowing in to this component is negative and anything flowing out of this component is positve (so the exact other way round)

**Comment by Bachibouzouk** [comment]
> this means in respect to a **component** that anything flowing in to this component is negative and anything flowing out of this component is positve (so the exact other way round)

Yes exactly
